src/mono/mono/mini/mini-arm64.h

```diff
@@ -52,7 +52,7 @@
 /* v16/v17 is reserved for a scratch reg */
 #define MONO_ARCH_CALLEE_FREGS 0xfffc00ff
 /* v8..v15 */
-#define MONO_ARCH_CALLEE_SAVED_FREGS 0xff00
+#define MONO_ARCH_CALLEE_SAVED_FREGS (0xff00 & ~(1 << 31))
 
```

Comment:
This change should effect MONO_ARCH_CALLEE_XREGS instead.

Reply:
I had to change `MONO_ARCH_CALLEE_SAVED_FREGS ` because of this assertion https://github.com/dotnet/runtime/blob/main/src/mono/mono/mini/mini-codegen.c#L147

Reply:
Now I map `NEON_TMP_REG` to `FP_TEMP_REG`. So this change is no longer needed.